examples/freertos_plus_tcp/z_sub_st.c

```diff
@@ -27,13 +27,17 @@
 #endif
 
 #define KEYEXPR "demo/example/**"
+#define N 2147483647  // max int value by default
```

Comment:
It's safer to use `int n = ~0;` in such a way we always have the guarantee `n` is `max(int)` regardless the actual size of `int`.